composer.json

```diff
@@ -40,7 +40,8 @@
         "lib/",
         "tests/SuiteCRM/",
         "tests/unit/lib/SuiteCRM/"
-      ]
+      ],
+      "Api\\": "Api/"
```

Comment:
Why do we need an new namespace here. Classes should be in the SuiteCRM namespace

Reply:
As I mentioned above, I've just made a working example. Let's talk about a possible merge between this and yours.